specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/capacityReservation.json

```diff
@@ -960,6 +971,22 @@
           "type": "string",
           "format": "date-time",
           "description": "Specifies the time at which the Capacity Reservation resource was created. Minimum api-version: 2021-11-01."
+        },
+        "scheduleProfile": {
+          "type": "object",
+          "properties": {
+            "start": {
+              "type": "string",
+              "format": "date-time",
+              "description": "The start date of the schedule in ISO 8601 format (DateTimeOffset). Only the date part is allowed; the time part will be defaulted to 12:00 PM UTC. Example: 2025-06-27."
```

Comment:
We are not accepting date time in ISO 8601 format (DateTimeOffset), pls. remove it.